roles/kubernetes/master/tasks/set-bind-addresses.yml

```diff
@@ -0,0 +1,4 @@
+- name: Set scheduler and controller manager bind addresses to host private IP
+  set_fact:
+    kube_scheduler_bind_address: "{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}"
```

Comment:
Won't this override the values set in the default role/

Reply:
I'm honestly not sure. Where should this ideally be placed?

Reply:
@waynr should I do something like
```
when: cloud_provider is defined and cloud_provider == 'openstack'
```
?

Or should I make a custom bool var for us specifically?

Reply:
Yeah I would go with your `when` idea.